internal/plugin/vault_config_experiments.go

```diff
@@ -0,0 +1,100 @@
+// Copyright (c) HashiCorp, Inc.
+// SPDX-License-Identifier: MPL-2.0
+
+package plugin
+
+import (
+	"fmt"
+
+	"github.com/hashicorp/terraform-plugin-go/tftypes"
+)
+
+type vaultExperimentsConfig struct {
```

Comment:
Given the values here, can't we use [*tfStringSlice](https://github.com/hashicorp-forge/terraform-provider-enos/blob/main/internal/plugin/tftypes.go#L418) instead of creating an entirely new type?

Example: https://github.com/hashicorp-forge/terraform-provider-enos/blob/main/internal/plugin/resource_vault_init.go#L40